Updates to sample documentation (#180)
aws-lumberyard-dev/o3de-multiplayersample: README.md

# MultiplayerSample Project

![MPS SplashScreen](Documentation/Media/splashscreen.png)

> A third-person multiplayer game sample for O3DE, where robots battle one another for dominance in a multi-tiered, under-construction, Starbase

Reviewer: Suggestion:
```suggestion
A third-person multiplayer game sample for O3DE where robots battle one another for dominance in a multi-tiered Starbase that is under construction.
```

> When debugging set `net_UdpTimeoutConnections` to false, this will prevent connection closures when stopped on breakpoints.

## Levels in this Project

This project ships with several levels, the ones of note are:

1. NewStarBase - The main game level, default to using this level
2. GamePlayTest - Everything needed for gameplay, but in a tiny, fast loading level. Gems, HUD, everything is there.
3. StartMenu - An example menu to enable joining, hosting and connecting to servers.
4. MultiplayerScriptingSample - An example of Multiplayer and scripting.

Other levels in the project are there for testing or performance evaluation purposes and should be considered experimental.

## How to contribute?

> The sample is managed by the O3DE special interest group SIG, [SIG/Network](https://github.com/o3de/sig-network).

> O3DE cannot work without the help and input from as many of its community members as possible. You do not need anyone’s permission to get involved and contribute to the project. The #sig-network channel on O3DE Discord is a great place to begin getting involved.

Reviewer: It would be nice to make these statements more positive. Maybe something like:
```suggestion
O3DE only works with the help and input from as many of its community members as possible. Anyone can get involved and contribute to the project, participation is welcome and encouraged. The #sig-network channel on O3DE Discord is a great place to begin getting involved.
```

> You can fix existing [issues](https://github.com/o3de/o3de-multiplayersample/issues) or backlog features or [make requests](https://github.com/o3de/o3de-multiplayersample/issues/new/choose) or proposals for new features.

Author: ```suggestion
You can report issues, fix known [issues](https://github.com/o3de/o3de-multiplayersample/issues or tackle backlogged features. In addition you can make [make requests](https://github.com/o3de/o3de-multiplayersample/issues/new/choose) and proposals for new features or extensions.
```
